packages/web/src/dev/ValidatorsListRow.tsx

```diff
@@ -0,0 +1,185 @@
+import * as React from 'react'
+import { Text as RNText, View } from 'react-native'
+import CopyToClipboard from 'src/dev/CopyToClipboard'
+import ProgressCutBar from 'src/dev/ProgressCutBar'
+import { styles } from 'src/dev/ValidatorsListStyles'
+import { I18nProps, withNamespaces } from 'src/i18n'
+import Chevron, { Direction } from 'src/icons/chevron'
+import { colors } from 'src/styles'
+import { cutAddress, formatNumber } from 'src/utils/utils'
+
+class Text extends RNText {
+  render() {
+    return <RNText style={[styles.defaultText, this.props.style]}>{this.props.children}</RNText>
+  }
+}
+
+interface Props {
+  group: any
```

Comment:
highly recommend typing this with am interface Group {address: string, name: string, ...etc}